src/js/main.js

```diff
@@ -0,0 +1,81 @@
+export const DiscourseRADParser = () => {
+  const radTabsElements = document.querySelectorAll('.rad-tabs-element');
+
+  radTabsElements.forEach((radTabsElement) => {
+    setDefaults(radTabsElement);
+    setupDropdowns(radTabsElement);
+  });
+
+  toggleElements();
+}
+
+const setDefaults = radTabsElement => {
+  const dropdowns = radTabsElement.querySelectorAll('select');
+
+  dropdowns.forEach((dropdown) => {
+    const name = dropdown.name;
+
+    const options = dropdown.querySelectorAll('option');
+    let defaultWasSet = false;
+    options.forEach((option) => {
+      if (localStorage.getItem(name) === option.value) {
+        defaultWasSet = true;
+        option.setAttribute('selected', 'selected');
+      }
+    });
+
+    if (!defaultWasSet) {
+      const firstOption = dropdown.firstElementChild;
+      firstOption.setAttribute('selected', 'selected');
```

Comment:
Isn't this what the browser gives use for free? What is the use-case?

Reply:
I think I was just paranoid about it. I guess I can remove it.